CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Add Twilio validation to make sure that post requests are coming from Twilio and relevant tests(CU-dgmfbv).
 
+### Updated
+- Updated existing tests to work with implementation of Twilio validation (CU-dgmfbv).
```

Comment:
No need to mention updating the tests in the changelog. In theory, every change will included updating tests :smile: